test/tests/log.js

```diff
@@ -0,0 +1,19 @@
+let exec = require('mz/child_process').exec;
+let getLog = require('../get_log');
```

Comment:
Maybe `getLog` doesn't need to be its own module and we can just move the multiple devices logging test into here?